integreat_cms/cms/utils/matomo_api_manager.py

```diff
@@ -249,18 +249,24 @@ async def get_visits_per_language_async(self, loop, query_params, languages):
                     self.fetch(
                         session,
                         **query_params,
-                        segment=f"pageUrl=@/{language.slug}/wp-json/;pageUrl!@/pages/",
+                        segment=f"pageUrl=@/{language.slug}/wp-json/extensions/v3/",
                     )
                 )
                 for language in languages
             ]
             # Create separate task to gather offline download hits
             tasks.append(
                 loop.create_task(
-                    self.fetch(session, **query_params, segment="pageUrl=@/pages/"),
+                    self.fetch(session, **query_params, segment="pageUrl=@/wp-json/extensions/v3/pages/"),
                 )
             )
-            # Create task for total visits in all languages
+            # Create separate task to gather online/web app download hits
+            tasks.append(
+                loop.create_task(
+                    self.fetch(session, **query_params, segment="pageUrl=@/wp-json/extensions/v3/children/"),
```

Comment:
I don't think the children endpoint is enough here... this will just cause a lot more inconsistencies since pages endpoint + children endpoint will not match the total number of all downloads.
I think we should invert the condition for offline downloads to get the number which we refer to as "online downloads"...

Reply:
Why not? We only exactly track these 2 endpoints. And the numbers do add up: 
![image](https://user-images.githubusercontent.com/15197000/198039825-8860bd30-5e7a-449d-b16b-91c245835cab.png)

Reply:
Ahh, I wasn't aware that the single page endpoint isn't used at all and all other endpoints are not tracked via Matomo. Alright, sorry, then it's fine.